src/lib.rs

```diff
@@ -94,50 +116,115 @@ fn cfg_vis_impl(
     cfg_vis_attr: CfgVisAttrArgs,
     mut item: syn::Item,
 ) -> syn::Result<proc_macro::TokenStream> {
-    fn proj_item(
-        item: &mut syn::Item,
-    ) -> syn::Result<(&mut syn::Visibility, &mut Vec<syn::Attribute>)> {
-        let proj = match item {
-            syn::Item::Const(i) => (&mut i.vis, &mut i.attrs),
-            syn::Item::Enum(i) => (&mut i.vis, &mut i.attrs),
-            syn::Item::ExternCrate(i) => (&mut i.vis, &mut i.attrs),
-            syn::Item::Fn(i) => (&mut i.vis, &mut i.attrs),
-            syn::Item::Macro2(i) => (&mut i.vis, &mut i.attrs),
-            syn::Item::Mod(i) => (&mut i.vis, &mut i.attrs),
-            syn::Item::Static(i) => (&mut i.vis, &mut i.attrs),
-            syn::Item::Struct(i) => (&mut i.vis, &mut i.attrs),
-            syn::Item::Trait(i) => (&mut i.vis, &mut i.attrs),
-            syn::Item::TraitAlias(i) => (&mut i.vis, &mut i.attrs),
-            syn::Item::Type(i) => (&mut i.vis, &mut i.attrs),
-            syn::Item::Union(i) => (&mut i.vis, &mut i.attrs),
-            syn::Item::Use(i) => (&mut i.vis, &mut i.attrs),
-            _ => {
-                return Err(syn::Error::new(
-                    item.span(),
-                    "`cfg_vis` can only apply on item with visibility",
-                ));
-            }
-        };
+    let __cfg_vis_accumulator_declare_path = __cfg_vis_accumulator_declare_path();
+
+    let (_, attrs) = proj_item(&mut item);
+    let mut accumulator: Vec<_> = attrs
+        .into_iter()
+        .filter(|attr| attr.path == __cfg_vis_accumulator_declare_path)
```

Comment:
Could direct `find` be used here?

Reply:
I want a compile error when there are more than one accumulators.